.github/workflows/wheel_manylinux_nightly.yaml

```diff
@@ -42,6 +42,13 @@ jobs:
         CUDA: ${{ matrix.config.cuda }}
       run: |
         docker/bash.sh --no-gpu $IMAGE ./wheel/build_wheel_manylinux.sh --cuda $CUDA
+    - name: Test
+      env:
+        IMAGE: ${{ matrix.config.image }}
+        WHEEL_TEST: "WHEEL_TEST"
+      continue-on-error: true
```

Comment:
```suggestion
```
do we still need this now that the skips are in place in apache/tvm?

Reply:
ah I see, yeah keeping it makes sense then. It'd be nicer to assert that no new tests broke but that seems like it'd be more complicated than it's worth here. The tests still error out complaining about `junitparser` missing before running though so that needs to be fixed before we can merge this